app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -22,7 +25,11 @@ class Identification extends React.Component {
       current: { ...this.props.currentState },
       errors: {},
       modified: {},
-      languageOptions: this.getLanguageOptions(),
+      primaryLanguageMessage: this.getPrimaryLanguageSupportMessage(tryToMatchLanguage(this.props.currentState.patient.primary_language)),
```

Comment:
Is there a reason this is stored in the state and can't just be called as a function using `selectedPrimaryLanguage`?

Reply:
It avoids unnecessary computation in `render()`. It is a "state"-ful part of the component, so I don't really see an issue with this. How strongly do you feel about having it be a function in render? I'm not necessarily against it, but why add computation if you don't need to?